charts/node-ipam-controller/templates/serviceaccount.yaml

```diff
@@ -32,3 +32,33 @@ subjects:
   - kind: ServiceAccount
     name: {{ include "node-ipam-controller.serviceAccountName" . }}
     namespace: {{ .Release.Namespace }}
+---
+{{- if .Values.leaderElection.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "node-ipam-controller.roleName" . }}
```

Comment:
We have multiple roles, the [convention](https://github.com/kubernetes-sigs/node-ipam-controller/blob/main/charts/node-ipam-controller/templates/serviceaccount.yaml#L15) is `{{ include "node-ipam-controller.serviceAccountName" . }}-XXX-role`

Reply:
yep! this one should be fixed now too!